server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
@chriscartlidge @matteofigus worth us all meeting up one afternoon to talk about this stuff? 🍺